test/e2e.test.ts

```diff
@@ -15,9 +15,9 @@ const readFile = (filename: string) => fs.readFileSync(filename, 'utf8')
 test('build fixture project', t => {
   process.chdir(fixtureDir)
 
-  exec('yarn install', { stdio: 'inherit' })
-  exec('yarn clean', { stdio: 'inherit' })
-  exec('yarn build', { stdio: 'inherit' })
+  exec('npx yarn install', { stdio: 'inherit' })
+  exec('npx yarn clean', { stdio: 'inherit' })
+  exec('npx yarn build', { stdio: 'inherit' })
```

Comment:
This is nonsense, yarn is a replacement for npm and npx.

Reply:
I didn't have yarn installed, so the tests fail. happy to change it back